### PR TITLE
chore(deps): update to Angular 13

### PR DESCRIPTION
**Pull request by BazylPL**

### CODE REVIEW

**Comment by BazylPL** [comment]
It needs to be tested on node >=16.10.0

**Comment by mwaldheim** [comment]
> It needs to be tested on node >=16.10.0

Node 14.17.0 works too

File: apps/playground/package.json [hunk 1]
```diff
@@ -2,27 +2,27 @@
   "name": "playground",
   "private": true,
   "dependencies": {
-    "@angular/common": "^12.0.4",
-    "@angular/compiler": "^12.0.4",
-    "@angular/core": "^12.0.4",
-    "@angular/platform-browser": "^12.0.4",
-    "@angular/platform-browser-dynamic": "^12.0.4",
-    "@angular/router": "^12.0.4",
+    "@angular/common": "^13.0.3",
```
Comment (mwaldheim) [inline apps/playground/package.json:5]: @BazylPL Can u try to Update it to 13.1.x?

**Comment by mwaldheim** [comment]
> It needs to be tested on node >=16.10.0

You need to change the Docker-images in .circleci/config.yml in Line 7, 24 and 41 to Version node:16.13 (the latest 16.x https://hub.docker.com/_/node?tab=tags&page=1&name=16.)

**Comment by mwaldheim** [comment]
Did you know, what angular-google-charts@0.0.0 postinstall makes? This command run on the Error now.

**Comment by mwaldheim** [comment]
now the Jest-Call is wrong :)

**Comment by BazylPL** [comment]
Yep, fixed the jest issue but there is a problem with some tests. I'll try to fix this.

**Comment by BazylPL** [comment]
I'm having a hard time fixing test issues. If someone knows how to deal with it, please contribute :)

**Comment by NashIlli** [comment]
+1
any update on this? Thanks!!

**Pull request closed without merging**